pkg/resource/hyper_parameter_tuning_job/manager_test_suite_test.go

```diff
@@ -100,15 +103,31 @@ func (d *testRunnerDelegate) Equal(a acktypes.AWSResource, b acktypes.AWSResourc
 	ac := a.(*resource)
 	bc := b.(*resource)
 	// Ignore LastTransitionTime since it gets updated each run.
-	opts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(ackv1alpha1.Condition{}, "LastTransitionTime")}
+	opts := []cmp.Option{cmpopts.EquateEmpty(),
+		cmpopts.IgnoreFields(ackv1alpha1.Condition{}, "LastTransitionTime"),
+		cmpopts.IgnoreFields(svcapitypes.HyperParameterTrainingJobSummary{}, "CreationTime"),
+		cmpopts.IgnoreFields(svcapitypes.HyperParameterTrainingJobSummary{}, "TrainingStartTime"),
+		cmpopts.IgnoreFields(svcapitypes.HyperParameterTrainingJobSummary{}, "TrainingEndTime")}
```

Comment:
Values that are coming from the server side should be ok to compare, why do we need to ignore them ?